assets/src/edit-story/components/library/panes/text/stories/texSet.js

```diff
@@ -0,0 +1,187 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import TextSet from '../textSets/textSet';
+import LibraryProvider from '../../../libraryProvider';
+import { UnitsProvider } from '../../../../../units';
+import { getBox } from '../../../../../units/dimensions';
+import { TransformProvider } from '../../../../transform';
+import { FontProvider } from '../../../../../app/font';
+import { APIProvider } from '../../../../../app/api';
+import { ConfigProvider } from '../../../../../app/config';
+import { PAGE_RATIO, TEXT_SET_SIZE } from '../../../../../constants';
+
+export default {
+  title: 'Stories Editor/Components/Library/Panes/Text/TextSet',
+  component: TextSet,
+  parameters: {
+    backgrounds: {
+      default: 'Dark',
+    },
+  },
+};
+
+export const _default = () => {
+  const textSet1 = [
+    {
+      opacity: 100,
+      flip: {
+        vertical: false,
+        horizontal: false,
+      },
+      rotationAngle: 0,
+      lockAspectRatio: true,
+      backgroundTextMode: 'NONE',
+      font: {
+        family: 'Alegreya',
+        service: 'fonts.google.com',
+        fallbacks: ['serif'],
+        weights: [400, 500, 700, 800, 900],
+        styles: ['regular', 'italic'],
+        variants: [
+          [0, 400],
+          [1, 400],
+          [0, 500],
+          [1, 500],
+          [0, 700],
+          [1, 700],
+          [0, 800],
+          [1, 800],
+          [0, 900],
+          [1, 900],
+        ],
+      },
```

Comment:
We should start including font metrics (metrics key found in fonts.json), so cropping will work.

Reply:
Will add, thanks!